Tools/TestWebKitAPI/Tests/IPC/IPCSerialization.mm

```diff
@@ -175,6 +178,40 @@ id valueAsID() const
     } };
 }
 
+#if PLATFORM(MAC)
+inline bool isEqual(WKDDActionContext *a, WKDDActionContext* b)
+{
+    auto ddResultRefEqual = [](DDResultRef a, DDResultRef b) {
+        if (a) {
+            if (!b)
+                return false;
+            if (!CFEqual(a, b))
```

Comment:
Doesn't CFEqual already handle the cases where one or both is null?

Reply:
You... are probably right lol.